### ISSUE TITLE
Expose Kafka broker RTT metrics as a system table

### ISSUE THREAD

**Issue by cirego** [MaterializeInc/materialize#6190]
With #5666 and #6131, we added various kafka metrics to a system table within Materialize, but
these metrics are per-partition/consumer. We also want to track RTT metrics and these are exposed
as a histogram on the Broker. This issue tracks creating a new table that will help customer
diagnose slow ingest by querying rtt metrics.

**Comment by cirego** [comment]
Here's a snapshot of the broker RTT metrics:

```
materialize=> select * from mz_kafka_broker_rtt;
    consumer_name    | source_id | dataflow_id |   broker_name    | rtt_min | rtt_max | rtt_avg | rtt_sum | rtt_cnt | rtt_stddev | rtt_hdrsize | rtt_p50 | rtt_p75 | rtt_p90 | rtt_p95 | rtt_p99 | rtt_p99_99 | rtt_outofrange 
---------------------+-----------+-------------+------------------+---------+---------+---------+---------+---------+------------+-------------+---------+---------+---------+---------+---------+------------+----------------
 rdkafka#consumer-3  | u1        |          30 | kafka:9092/1     |  500543 |  500911 |  500727 | 1001454 |       2 |          0 |       13424 |  501759 |  501759 |  501759 |  501759 |  501759 |     501759 |              0
 rdkafka#consumer-4  | u1        |          30 | kafka:9092/1     |  999956 | 1001408 | 1000682 | 2001364 |       4 |          0 |       26848 | 1003518 | 1003518 | 1003518 | 1003518 | 1003518 |    1003518 |              0
 rdkafka#consumer-5  | u1        |          30 | kafka:9092/1     | 1000012 | 1000798 | 1000404 | 2000810 |       4 |          0 |       26848 | 1003518 | 1003518 | 1003518 | 1003518 | 1003518 |    1003518 |              0
 rdkafka#consumer-6  | u1        |          30 | kafka:9092/1     |  500196 |  501255 |  500725 | 1001451 |       2 |          0 |       13424 |  501759 |  501759 |  501759 |  501759 |  501759 |     501759 |              0
 rdkafka#consumer-7  | u1        |          30 | kafka:9092/1     | 1001212 | 1001992 | 1001602 | 2003204 |       4 |          0 |       26848 | 1003518 | 1003518 | 1003518 | 1003518 | 1003518 |    1003518 |              0
 rdkafka#consumer-8  | u1        |          30 | kafka:9092/1     |  500494 |  501021 |  500757 | 1001515 |       2 |          0 |       13424 |  501759 |  501759 |  501759 |  501759 |  501759 |     501759 |              0
 rdkafka#consumer-9  | u1        |          30 | kafka:9092/1     | 1000572 | 1003462 | 1002016 | 2004034 |       4 |          0 |       26848 | 1003518 | 1003518 | 1003518 | 1003518 | 1003518 |    1003518 |              0
 rdkafka#consumer-10 | u1        |          30 | kafka:9092/1     | 1001142 | 1002340 | 1001740 | 2003482 |       4 |          0 |       26848 | 1003518 | 1003518 | 1003518 | 1003518 | 1003518 |    1003518 |              0
 rdkafka#consumer-11 | u1        |          30 | kafka:9092/1     |  999826 | 1001120 | 1000472 | 2000946 |       4 |          0 |       26848 | 1003518 | 1003518 | 1003518 | 1003518 | 1003518 |    1003518 |              0
 rdkafka#consumer-12 | u1        |          30 | kafka:9092/1     |  999952 | 1001192 | 1000572 | 2001144 |       4 |          0 |       26848 | 1003518 | 1003518 | 1003518 | 1003518 | 1003518 |    1003518 |              0
 rdkafka#consumer-13 | u1        |          30 | kafka:9092/1     |  500347 |  502202 |  501274 | 1002549 |       2 |       1024 |       13424 |  501759 |  503807 |  503807 |  503807 |  503807 |     503807 |              0
 rdkafka#consumer-14 | u1        |          30 | kafka:9092/1     |  500384 |  501008 |  500696 | 1001392 |       2 |          0 |       13424 |  501759 |  501759 |  501759 |  501759 |  501759 |     501759 |              0
 rdkafka#consumer-15 | u1        |          30 | kafka:9092/1     | 1000132 | 1001344 | 1000738 | 2001476 |       4 |          0 |       26848 | 1003518 | 1003518 | 1003518 | 1003518 | 1003518 |    1003518 |              0
 rdkafka#consumer-16 | u1        |          30 | kafka:9092/1     |  500549 |  500757 |  500653 | 1001306 |       2 |          0 |       13424 |  501759 |  501759 |  501759 |  501759 |  501759 |     501759 |              0
 rdkafka#consumer-17 | u1        |          30 | kafka:9092/1     | 1001420 | 1001520 | 1001470 | 2002940 |       4 |          0 |       26848 | 1003518 | 1003518 | 1003518 | 1003518 | 1003518 |    1003518 |              0
 rdkafka#consumer-18 | u1        |          30 | kafka:9092/1     |  999856 | 1001352 | 1000604 | 2001208 |       4 |          0 |       26848 | 1003518 | 1003518 | 1003518 | 1003518 | 1003518 |    1003518 |              0
 rdkafka#consumer-19 | u1        |          30 | kafka:9092/1     | 1000316 | 1001074 | 1000694 | 2001390 |       4 |          0 |       26848 | 1003518 | 1003518 | 1003518 | 1003518 | 1003518 |    1003518 |              0
 rdkafka#consumer-20 | u1        |          30 | kafka:9092/1     | 1001250 | 1001860 | 1001554 | 2003110 |       4 |          0 |       26848 | 1003518 | 1003518 | 1003518 | 1003518 | 1003518 |    1003518 |              0
 rdkafka#consumer-3  | u1        |          30 | GroupCoordinator |       0 |       0 |       0 |       0 |       0 |          0 |       13424 |       0 |       0 |       0 |       0 |       0 |          0 |              0
 rdkafka#consumer-4  | u1        |          30 | GroupCoordinator |       0 |       0 |       0 |       0 |       0 |          0 |       26848 |       0 |       0 |       0 |       0 |       0 |          0 |              0
 rdkafka#consumer-5  | u1        |          30 | GroupCoordinator |       0 |       0 |       0 |       0 |       0 |          0 |       26848 |       0 |       0 |       0 |       0 |       0 |          0 |              0
 rdkafka#consumer-6  | u1        |          30 | GroupCoordinator |       0 |       0 |       0 |       0 |       0 |          0 |       13424 |       0 |       0 |       0 |       0 |       0 |          0 |              0
 rdkafka#consumer-7  | u1        |          30 | GroupCoordinator |       0 |       0 |       0 |       0 |       0 |          0 |       26848 |       0 |       0 |       0 |       0 |       0 |          0 |              0
 rdkafka#consumer-8  | u1        |          30 | GroupCoordinator |       0 |       0 |       0 |       0 |       0 |          0 |       13424 |       0 |       0 |       0 |       0 |       0 |          0 |              0
 rdkafka#consumer-9  | u1        |          30 | GroupCoordinator |    4682 |    4682 |    4682 |    4682 |       2 |          0 |       26848 |    4702 |    4702 |    4702 |    4702 |    4702 |       4702 |              0
 rdkafka#consumer-10 | u1        |          30 | GroupCoordinator |    5534 |    5534 |    5534 |    5534 |       2 |          0 |       26848 |    5534 |    5534 |    5534 |    5534 |    5534 |       5534 |              0
 rdkafka#consumer-11 | u1        |          30 | GroupCoordinator |       0 |       0 |       0 |       0 |       0 |          0 |       26848 |       0 |       0 |       0 |       0 |       0 |          0 |              0
 rdkafka#consumer-12 | u1        |          30 | GroupCoordinator |       0 |       0 |       0 |       0 |       0 |          0 |       26848 |       0 |       0 |       0 |       0 |       0 |          0 |              0
 rdkafka#consumer-13 | u1        |          30 | GroupCoordinator |       0 |       0 |       0 |       0 |       0 |          0 |       13424 |       0 |       0 |       0 |       0 |       0 |          0 |              0
 rdkafka#consumer-14 | u1        |          30 | GroupCoordinator |       0 |       0 |       0 |       0 |       0 |          0 |       13424 |       0 |       0 |       0 |       0 |       0 |          0 |              0
 rdkafka#consumer-15 | u1        |          30 | GroupCoordinator |       0 |       0 |       0 |       0 |       0 |          0 |       26848 |       0 |       0 |       0 |       0 |       0 |          0 |              0
 rdkafka#consumer-16 | u1        |          30 | GroupCoordinator |       0 |       0 |       0 |       0 |       0 |          0 |       13424 |       0 |       0 |       0 |       0 |       0 |          0 |              0
 rdkafka#consumer-17 | u1        |          30 | GroupCoordinator |       0 |       0 |       0 |       0 |       0 |          0 |       26848 |       0 |       0 |       0 |       0 |       0 |          0 |              0
 rdkafka#consumer-18 | u1        |          30 | GroupCoordinator |       0 |       0 |       0 |       0 |       0 |          0 |       26848 |       0 |       0 |       0 |       0 |       0 |          0 |              0
 rdkafka#consumer-19 | u1        |          30 | GroupCoordinator |       0 |       0 |       0 |       0 |       0 |          0 |       26848 |       0 |       0 |       0 |       0 |       0 |          0 |              0
 rdkafka#consumer-20 | u1        |          30 | GroupCoordinator |       0 |       0 |       0 |       0 |       0 |          0 |       26848 |       0 |       0 |       0 |       0 |       0 |          0 |              0
(36 rows)
```

@krishmanoh2, how does this look to you?

**Comment by cirego** [comment]
Fixed by #6202.
